src/user.js

```diff
@@ -7,11 +7,17 @@ class User {
 
   }
 
-  changeFavorites() {
-
+  addToFavorites(recipe) {
+    this.favoriteRecipes.push(recipe);
+  }
+  
+  removeFromFavorites(recipe) {
+    const i = this.favoriteRecipes.indexOf(recipe);
+    this.favoriteRecipes.splice(i, 1)
```

Comment:
I like this change, it's more explicit as to what is actually happening.  Follows the SRP, nice job.